src/throttlePromise.ts

```diff
@@ -15,7 +15,7 @@ type Queue = {
 
 interface ISbThrottle {
   abort: () => any
-  (args: []): Promise<Queue>
+  (...args: any): Promise<Queue>
```

Comment:
Non-blocking: @alexjoverm  Can we do anything to avoid using `any` here? I will leave it to your personal criteria if it's something we should do on the scope of this PR or in the v7 refactor. In normal cases I would block a PR from merging if there are `any` on the types